tests/benchmark/Dockerfile.in

```diff
@@ -1,6 +1,15 @@
-FROM ${FULL_BASE_IMAGE}
+FROM gcr.io/google-appengine/python:latest
 
-RUN hg clone -r 9923b81a1d34 https://hg.python.org/benchmarks /app/benchmarks
-WORKDIR /app/benchmarks
-RUN python perf.py -r -b default /usr/bin/python2.7 /usr/bin/python3.4
-RUN python perf.py -r -b default /usr/bin/python3.4 /opt/python3.5/bin/python3.5
+# Install performance
+RUN python2 -m pip install performance
+
+# Create virtual environment
+RUN python2 -m pip install -U virtualenv
+
+# Download ensurepip module which prevents the failure when benchmarking python3
```

Comment:
Below is the error message:

Step 4 : RUN pyperformance run --debug-single-value --python=python3.4 -o py3.4.json && pyperformance run --debug-single-value --python=python2.7 -o py2.7.json && pyperformance run --debug-single-value --python=python3.5 -o py3.5.json && pyperformance compare py2.7.json py3.4.json --output_style table && pyperformance compare py3.4.json py3.5.json --output_style table
 ---> Running in ddb9875f6698
Creating the virtual environment venv/cpython3.4-bffe9c063051
Execute: /usr/bin/python3.4 -m venv --without-pip venv/cpython3.4-bffe9c063051
Execute: venv/cpython3.4-bffe9c063051/bin/python -c 'import sys; print(sys.hexversion)'
Python hexversion: 30402f0
Execute: venv/cpython3.4-bffe9c063051/bin/python -m ensurepip --verbose
/home/vmagent/app/venv/cpython3.4-bffe9c063051/bin/python: No module named ensurepip
Command venv/cpython3.4-bffe9c063051/bin/python -m ensurepip --verbose failed with exit code 1
Download https://bootstrap.pypa.io/get-pip.py into venv/get-pip.py

Remove directory venv/cpython3.4-bffe9c063051
Traceback (most recent call last):
  File "/usr/local/bin/pyperformance", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/performance/cli.py", line 226, in main
    _main()
  File "/usr/local/lib/python2.7/dist-packages/performance/cli.py", line 206, in _main
    exec_in_virtualenv(options)
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 551, in exec_in_virtualenv
    venv.create()
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 540, in create
    self._create_venv()
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 454, in _create_venv
    ok = self._create_venv_cmd(cmd + [venv_path], install_pip)
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 432, in _create_venv_cmd
    ok = self.install_pip()
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 398, in install_pip
    download(GET_PIP_URL, filename)
  File "/usr/local/lib/python2.7/dist-packages/performance/venv.py", line 211, in download
    with response:
AttributeError: addinfourl instance has no attribute '__exit__'
The command '/bin/sh -c pyperformance run --debug-single-value --python=python3.4 -o py3.4.json && pyperformance run --debug-single-value --python=python2.7 -o py2.7.json && pyperformance run --debug-single-value --python=python3.5 -o py3.5.json && pyperformance compare py2.7.json py3.4.json --output_style table && pyperformance compare py3.4.json py3.5.json --output_style table' returned a non-zero code: 1
ERROR
ERROR: build step "gcr.io/cloud-builders/docker@sha256:83b9c74c2440d2d5a3d9faa40536f0a1b63b424172e83cdf57ffb7f0237e6fa2" failed: exit status 1